tests/virt_autotest/virt_utils.pm

```diff
@@ -55,6 +55,14 @@ sub enable_debug_logging {
     }
     save_screenshot;
 
+    # enable qemu core dumps
```

Comment:
Can you put the qemu core dump related code into a separete PR? It should be for a completely different purpose from SR-IOV automation commit. We can merge the two commits in this single PR.

Reply:
the coredump was committed in multiple commits. And the verification need much effort as it needs the entire installation and actural coredumps. If I try to seperate it from other commits by picking the codes manually, it may have risk to some extend.